src/shared/parse-argument.c

```diff
@@ -163,3 +164,10 @@ int parse_machine_argument(const char *s, const char **ret_host, BusTransport *r
 
         return 0;
 }
+
+int parse_background_argument(const char *s, char **arg) {
+        if (!isempty(s) && !looks_like_ansi_color_code(s))
+                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Invalid background color: %s", s);
```

Comment:
Please explicitly mention the option name, e.g.
```
"Failed to validate --background= argument: %s"
```
See log messages in other parsers.